apps/docs-website/tsconfig.json

```diff
@@ -9,5 +9,8 @@
     "incremental": true,
     "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
     "noUncheckedSideEffectImports": true
+  },
+  "mdx": {
+    "checkMdx": true
```

Comment:
👎

Reply:
This typechecks the mdx files. Is there an issue with that?

Reply:
i cannot find any documentation about this

Reply:
the addition is fine then, but i think we should add a comment with this link, right here